plugins/woocommerce/src/Admin/API/Options.php

```diff
@@ -223,6 +223,7 @@ public static function get_default_option_permissions() {
 			'woocommerce_private_link',
 			'woocommerce_share_key',
 			'woocommerce_show_lys_tour',
+			'woocommerce_schema_version',
```

Comment:
Out of curiosity, why do we need this change?

Reply:
I had to add this here in order for it to have edit permission through WC Test Helper, so that the testing instructions can be executed.

Reply:
Oh ok, then we could remove this and not commit it - instead we could update the value with wp cli (that's how I actually did it)

`wp option update woocommerce_schema_version 430`

Reply:
Good call! I'll update the testing instructions.